Command db:console using --pass as password parameter

Hi! 
I noticed the db:console command is trying to connect to mysql cli using --pass param instead of --password which throws: 

`mysql: [ERROR] unknown variable 'pass=mypassword*'`

--- a/src/N98/Util/Console/Helper/DatabaseHelper.php
+++ b/src/N98/Util/Console/Helper/DatabaseHelper.php
@@ -212,7 +212,7 @@ class DatabaseHelper extends AbstractHelper
             . (isset($this->dbSettings['port'])
                 ? '-P' . escapeshellarg($this->dbSettings['port']) . ' ' : '')
             . (strlen($this->dbSettings['password'])
-                ? '--pass=' . escapeshellarg($this->dbSettings['password']) . ' ' : '')
+                ? '--password=' . escapeshellarg($this->dbSettings['password']) . ' ' : '')
             . escapeshellarg($this->dbSettings['dbname']);
 
         return $string;

